packages/firestore/test/integration/api/composite_index_query.test.ts

```diff
@@ -136,4 +149,157 @@ apiDescribe('Composite Index Queries', persistence => {
       });
     });
   });
+
+  describe('Aggregation queries - sum / average', () => {
+    it('aggregate query supports collection groups - multi-aggregate', () => {
+      const testHelper = new CompositeIndexTestHelper();
+      return testHelper.withTestCollection(persistence, async (coll, db) => {
+        const collectionGroupId = coll.id;
+        const docPaths = [
+          `${collectionGroupId}/cg-doc1`,
+          `abc/123/${collectionGroupId}/cg-doc2`,
+          `zzz${collectionGroupId}/cg-doc3`,
+          `abc/123/zzz${collectionGroupId}/cg-doc4`,
+          `abc/123/zzz/${collectionGroupId}`
+        ];
+        const batch = writeBatch(db);
+        for (const docPath of docPaths) {
+          // Add test specific fields to the document value
+          batch.set(
+            doc(db, docPath),
+            testHelper.addTestSpecificFieldsToDoc({ a: 2 })
+          );
+        }
+        await batch.commit();
+        const snapshot = await getAggregateFromServer(
+          testHelper.query(collectionGroup(db, collectionGroupId)),
+          {
+            count: count(),
+            sum: sum('a'),
+            avg: average('a')
+          }
+        );
+        expect(snapshot.data().count).to.equal(2);
+        expect(snapshot.data().sum).to.equal(4);
+        expect(snapshot.data().avg).to.equal(2);
+      });
+    });
+
+    it('performs aggregations on documents with all aggregated fields using getAggregationFromServer', () => {
```

Comment:
I have changed the field name from "x" to "a", so no new fields introduced. 
But since this is a CollectionGroup query, a new index is required after all. As long as any new tests on collection group query will keep using field "a", then we have achieved the re-use and limit index numbers.

Reply:
I was thinking about the non-collection-group indexes. But after comparing the new indexes to the existing ones, what you said still applies, we'd still have to create new indexes even if we changed field names.